src/manpageContentProvider.ts

```diff
@@ -181,4 +185,7 @@ export async function openManPage(input: string, inActiveColumn: boolean): Promi
         viewColumn: inActiveColumn ? vscode.ViewColumn.Active : vscode.ViewColumn.Beside
     });
     textDocument.options.lineNumbers = 0; // off
+
+    const enableSyntaxHighlighting = (vscode.workspace.getConfiguration('manpages').get('enableSyntaxHighlighting', true) as boolean);
+    if(enableSyntaxHighlighting) vscode.languages.setTextDocumentLanguage(doc, 'manpage');
```

Comment:
That does work quite well and the syntax is as expected - but it seems that if we follow links to open a manpage this has to be done there as well (as of version 0.2.0 links work, but they are highlighted as "text-only", so a user has to manually adjust the syntax).

Ideally, this option can be dropped here and instead there a language detector be implemented - see #39.
As an alternative (or shortcut for now) - at the place where we open links do the same as those two lines.